examples/demo/site/modules/session_auth/files/js/roc_basic_auth.js

```diff
@@ -0,0 +1,325 @@
+var ROC_AUTH = ROC_AUTH || { };
+
+var loginURL = "/basic_auth_login";
```

Comment:
I guess here we should have an equivalent of "$site_url" .
At least add a FIXME.